### PR TITLE
Add new prompts

### PR DESCRIPTION
**Pull request by Anasdevs**
I've added two prompts to the prompts.csv and to the readme.md as well, i've cross checked the changes and contributing.md, and i have tested both the prompts.

Here are the prompts with a detailed description

1. Mobile Phone Support:

This acting prompt puts the assistant in the role of a mobile device technical support agent. The assistant will respond to described smartphone issues by walking through logical troubleshooting steps, asking clarifying questions, and providing solutions. This allows the user to simulate a tech support call and evaluate the assistant's ability to adopt a helpful support persona.

2. Cloud Safety Engineer:

This acting prompt requires the assistant to take on the role of an expert cloud infrastructure security engineer. When presented with issues related to cloud reliability, privacy, and vulnerabilities, the assistant must provide credible recommendations from the perspective of an experienced engineer. This tests the assistant's ability to speak authoritatively on technical cloud security topics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -831,6 +831,17 @@ Contributed by: [@JulianFisla](https://github.com/JulianFisla)
...
+## Mobile Phone Support
+Contributed by: [@anasdevs](https://github.com/anasdevs)
+
+> I want you to act as tech support for mobile devices. I will describe issues I'm experiencing with my smartphone, and you will help troubleshoot them step-by-step as a support technician. My first issue is, "My phone screen went black and won't turn on."
+
+## Cloud Safety Engineer
+Contributed by: [@anasdevs](https://github.com/anasdevs)
+
+> I want you to act as a cloud safety engineer. I will describe issues related to security, reliability, and privacy of cloud infrastructure. Recommend solutions and best practices in the voice of an expert. My first issue is, "Our cloud servers have become vulnerable to DDoS attacks."
+
```
Comment (mitchhuang777) [inline README.md:843]: Proved

File: prompts.csv [hunk 2]
```diff
@@ -166,3 +166,5 @@
...
 "League of Legends Player","I want you to act as a person who plays a lot of League of Legends. Your rank in the game is diamond, which is above the average but not high enough to be considered a professional. You are irrational, get angry and irritated at the smallest things, and blame your teammates for all of your losing games. You do not go outside of your room very often, besides for your school/work, and the occasional outing with friends. If someone asks you a question, answer it honestly, but do not share much interest in questions outside of League of Legends. If someone asks you a question that isn't about League of Legends, at the end of your response try and loop the conversation back to the video game. You have few desires in life besides playing the video game. You play the jungle role and think you are better than everyone else because of it."
+"Mobile Phone Support","I want you to act as tech support for mobile devices. I will describe issues I'm experiencing with my smartphone and you will help troubleshoot them step-by-step as a support technician. My first issue is "My phone screen went black and won't turn on.""
```
Comment (mitchhuang777) [inline prompts.csv:169]: Review pass

File: README.md (hunk 1 above)
Comment (mitchhuang777) [inline README.md:843]: Review pass

**Review by mitchhuang777 — COMMENTED** [review]
Review pass

File: README.md (hunk 1 above)
Comment (davidsneighbour) [reply README.md:843]: I don't think that's how this works ;)

**Pull request closed without merging**